Jellyfin.Api/Auth/BaseAuthorizationHandler.cs

```diff
@@ -57,6 +58,12 @@ public abstract class BaseAuthorizationHandler<T> : AuthorizationHandler<T>
                 return false;
             }
 
+            // UserId of Guid.Empty means token is an apikey.
+            if (userId.Equals(Guid.Empty))
```

Comment:
I would rather we add ApiKey = {true, false} to AuthorizationInfo and add a new claim in the Identity for ApiKey